CSS/adminStyle.css

```diff
@@ -0,0 +1,93 @@
+* {
+    box-sizing: border-box;
+    font-family: 'Dosis', sans-serif;
+}
+
+h1 {
+    color: #F75A5C;
+    width: 50%;
+    margin: 0 auto;
+    text-align: center;
+    padding-bottom: 40px;
+}
+
+.section_break{
+    border-top-style: dotted;
+    border-top-color: #F75A5C;
+    width: 100%;
+    margin-bottom: 60px;
+}
+
+thead tr th {
+    color: #2C3539;
+}
+
+/****** Add new entry table *****/
+
+.newProject{
+    width: 100%;
+    text-align:end;
+    margin-top: 20px;
+}
+
+.newProject th {
+    text-align: right;
+    padding-right: 70px;
+}
+
+.createNew {
+    background-color: #00aced;
+    border: none;
+    color: white;
+    padding: 16px 32px;
+    text-decoration: none;
+    margin: 4px 2px;
+    cursor: pointer;
+    font-size: 1.25rem;
+    border-radius: 20px;
+    text-align: right;
+}
+
+/***** Project Table */
+.projectsContainer {
+    color: #F75A5C;
+    width: 100%;
+}
+
+.projectsContainer tbody td {
+    padding-left: 5px;
+}
+
+th {
+    text-align:left;
+    padding-left: 5px;
+}
+
+input.edit {
```

Comment:
.edit and .delete are identical apart from the background colour so it would be better to combine these into one class and break out the background-color into 2 classes